plugins/org.wso2.developerstudio.humantaskeditor/HumanTaskEditor/hteditor.js

```diff
@@ -237,15 +244,33 @@ function generateTasks() {
 }
 
 /*
+ * Signature: addTask(){...}
+ * 
+ * This method generates creates a new task. This method add a dummy task node into the ht file and adds a new task tab and Ui elements respectively
+ * 
+ * 
+ */
+function deleteTask(taskNode) { //createFile
```

Comment:
Fixed in https://github.com/wso2/devstudio-tooling-bps/commit/1b5789567f9361a4df08ca65bf24fd1b1f132751